SkillsTraining/Assets/Scripts/OpenAIApi.cs

```diff
@@ -0,0 +1,34 @@
+using Azure;
+using Azure.AI.OpenAI;
```

Comment:
We will need to get the Azure.AI.OpenAI Unity package into the project before merging this or it won't compile and will block the Unity project from loading/working